recipe/build.sh

```diff
@@ -1,14 +1,15 @@
 #!/bin/bash
 
-autoreconf -fiv
+mkdir build_libjpeg && cd  build_libjpeg
 
-# Set here any HW specific flags (usually at CFLAGS)
-./configure --prefix=$PREFIX \
-    --enable-shared=yes \
-    --enable-static=yes \
-    --with-jpeg8 \
-    CFLAGS="-O3 -fPIC" \
-    NASM=yasm
-make
-make check
-make install
+cmake -D CMAKE_INSTALL_PREFIX=$PREFIX \
+      -D CMAKE_BUILD_TYPE=Release \
+      -D ENABLE_STATIC=1 \
+	  -D ENABLE_SHARED=1 \
+	  -D WITH_JPEG8=1 \
+	  -D NASM=yasm \
```

Comment:
I think here we need:
  -DCMAKE_ASM_NASM_COMPILER=yasm                                   \

Reply:
Also we probably need
-DCMAKE_INSTALL_LIBDIR="lib"                                     \

Reply:
Let me remove that...

Reply:
> Also we probably need
> -DCMAKE_INSTALL_LIBDIR="lib" \


Things seems to work without it. I'll double check it locally, but I guess it should be `$PREFIX/lib` instead of `"lib"`.